hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/security/token/block/BlockTokenSecretManager.java

```diff
@@ -226,7 +228,7 @@ public synchronized void addKeys(ExportedBlockKeys exportedKeys)
     if (isMaster || exportedKeys == null) {
       return;
     }
-    LOG.info("Setting block keys. BlockPool = {} .", blockPoolId);
+    LOG.info("Setting block keys. BlockPool = {} . NamenodeIndex = {} .", blockPoolId, nnIndex);
```

Comment:
I don't think nnIndex is what you want here. If you are looking for information around which nn this request came from, I would instead pass the BPServiceActor in from the BPOfferService. Then you can print out the nnAddr and the HAServiceState along with the log message.

Reply:
Decided not to pass BPServiceActor due to it being a private class. Namenode address will instead be logged when dn received command from standby or active nn